nexus/external-api/src/lib.rs

```diff
@@ -227,7 +227,13 @@ const PUT_UPDATE_REPOSITORY_MAX_BYTES: usize = 4 * GIB;
                 external_docs = {
                     url = "http://docs.oxide.computer/api/system-silos"
                 }
-            }
+            },
+            "system/update" = {
+                description = "Endpoints for uploading and managing system updates",
```

Comment:
This is a nit because I have no idea where this text actually gets displayed, but all the tags are about endpoints. I would say "Upload and manage system updates."